base/Plain_netCDF_Time.F90

```diff
@@ -511,3 +514,239 @@ subroutine convert_twostring_2_esmfinterval(symd, shms, interval, rc)
   end subroutine convert_twostring_2_esmfinterval
 
 end module Plain_NetCDF_Time
+
+
+module Fortran_read_file
```

Comment:
This name is too generic.   

Also needs standard namespace protection in module name.   `mapl_`

Reply:
I changed the module name to MAPL_scan_pattern_in_file.